scripts/compile.ts

```diff
@@ -1,25 +1,23 @@
-import { promises as filesystem } from 'fs'
-import * as path from 'path'
 import { CompilerOutputContract } from 'solc'
```

Comment:
Why are there changes to this file? I would try to really stick to only the necessary changes

Reply:
Fixed by https://github.com/protofire/safe-singleton-factory/commit/51f7591668b4a85b6b5c4d693712375258225699